src/Autowhatever.js

```diff
@@ -281,7 +283,7 @@ export default class Autowhatever extends Component {
 
     return (
       <div {...theme(`react-autowhatever-${id}-container`, 'container', isOpen && 'containerOpen')}>
-        <input {...inputProps} />
+        {this.props.renderInput(inputProps)}
```

Comment:
Just wondering what's the point to pass back `inputProps` to the consumer if they already have it? Can it just be `this.props.renderInput()` ?